cmd/kyma/alpha/deploy/opts_test.go

```diff
@@ -52,4 +53,35 @@ func TestOptsValidation(t *testing.T) {
 		err := opts.validateFlags()
 		require.NoError(t, err)
 	})
+	t.Run("Check workspace folder is not deleted when it is set", func(t *testing.T) {
+		ws := path.Join("testdata", "dummyWS")
+		Opts := Options{Source: "main", WorkspacePath: ws}
+		err := os.Mkdir(ws, 0700)
+		require.NoError(t, err)
+		wsp, err := Opts.ResolveLocalWorkspacePath()
+		require.NoError(t, err)
+		require.Equal(t, ws, wsp)
+		err = Opts.pathExists(ws, "dummy ws path")
```

Comment:
It will always fail, since you delete the path in ResolveLocalWorkspace. This is the exact same test as `Check kyma workspace is being deleted`
In my opinion this is a intended behaviour, please adapt or remove test

Reply:
Here I am cleaning up the folder I created. I think `require.NoError(t, err)` is confusing. Here the idea is to check `workspace` is not being deleted

Reply:
As discussed in meeting, `main` needs to be changed to `local`